cms/jinja2/templates/pages/cookies.html

```diff
@@ -0,0 +1,192 @@
+                                },
+                                {
+                                    "id": "off-1",
+                                    "label": {
+                                        "text": radio_off
+                                    },
+                                    "value": "off"
+                                }
+                            ]
+                        }) }}
+                </div>
+                {# fmt:on #}
+                <p>{{ _("Your preference selection will expire after one year.") }}</p>
+
+                <h3>{{ _("Cookies that help with our communications") }}</h3>
+                <p>
+                    {{ _("Some of our pages may contain content from other sites, like YouTube and Vimeo. We allow third party
+                    sites, like YouTube and Vimeo, to set cookies on our website when you view this content, they will
+                    measure how you engage with the content on our website.") }}
+                </p>
+
+                {% set campaigns_legend = _("Do you want to allow third party usage tracking?") %}
+                {# fmt:off #}
+                <div class="ons-u-mb-s">
+                    {{ onsRadios({
+                            "id": "cookies-measure-website-comms",
+                            "name": "cookies-campaigns",
+                            "legend": campaigns_legend,
+                            "radios": [
+                                {
+                                    "id": "on-2",
+                                    "label": {
+                                        "text": radio_on
+                                    },
+                                    "value": "on"
+                                },
+                                {
+                                    "id": "off-2",
+                                    "label": {
+                                        "text": radio_off
+                                    },
+                                    "value": "off"
+                                }
+                            ]
+                        }) }}
+                </div>
+                {# fmt:on #}
+
+                <p>{{ _("Your preference selection will expire after one year.") }}</p>
+                <p>
+                    {{ _("In addition, if you share a link to ons.gov.uk page, the service you share it on may set a cookie.
+                    We have no control over cookies set on other websites – you can turn them off, but not through us.") }}
+                </p>
+
+                <h3>{{ _("Cookies that remember your preference settings to tailor your experience") }}</h3>
+                <p>
+                    {{ _("These cookies remember the choices you make when using the site to personalise your experience.
+                    These cookies do not currently collect anydata and but are set in preparation for potential future
```

Comment:
Potential typo
```suggestion
                    These cookies do not currently collect any data and but are set in preparation for potential future
```

Reply:
Fixed in [46550b4](https://github.com/ONSdigital/dis-wagtail/pull/375/commits/46550b468ae6cf9c018055544c8f3ef0cb56378d)